
Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-credential-initializer-t9785[0m
{"level":"warn","ts":1597689065.8703783,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1597689065.8707128,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-working-dir-initializer-97k4r[0m
{"level":"warn","ts":1597689066.838989,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1597689066.840048,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-place-tools[0m

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-create-dir-workspace-xr76m[0m
{"level":"warn","ts":1597689071.9519916,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1597689071.9529243,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-git-source-rajattyagipvr-istio-master-r5dbn-frmvz[0m
{"level":"warn","ts":1597689072.1782458,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1597689074.351325,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/*************/istio.git @ v0.0.24 in path /workspace/source"}

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:
WARNING: no SHAs to merge, falling back to initial cloned commit

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-next-version[0m
created new version: 0.0.25 and written to file: ./VERSION
Tag v0.0.25 created and pushed to remote origin

Showing logs for build [32mrajattyagipvr-istio-master-r5dbn-1[0m stage [32msetup[0m and container [32mstep-tag-version[0m
error: Have you set up a git credential helper? See https://help.github.com/articles/caching-your-github-password-in-git/
: git output: To https://github.com/*************/istio.git
 ! [rejected]        v0.0.25 -> v0.0.25 (already exists)
error: failed to push some refs to 'https://github.com/*************/istio.git'
hint: Updates were rejected because the tag already exists in the remote.: failed to run 'git push origin v0.0.25' command in directory '', output: 'To https://github.com/*************/istio.git
 ! [rejected]        v0.0.25 -> v0.0.25 (already exists)
error: failed to push some refs to 'https://github.com/*************/istio.git'
hint: Updates were rejected because the tag already exists in the remote.'
[31m
Pipeline failed on stage 'setup' : container 'step-tag-version'. The execution of the pipeline has stopped.[0m
